lib/discordrb/data/server.rb

```diff
@@ -590,10 +590,8 @@ def max_emoji
     # @param limit [Integer] The maximum number of members between 1-1000 to return. Returns 1 member by default.
     # @return [Array<Member>] An array of member objects that match the given parameters.
     def search_members(name:, limit: nil)
-      response = JSON.parse(API::Server.search_guild_members(@bot.token, @id, name, limit))
-      return nil if response.empty?
```

Comment:
We're changing the method signature here so I don't want this getting merged in until 4.0, can we split this out?